libsrc/pce/psg.s

```diff
@@ -0,0 +1,29 @@
+
+        .include        "pce.inc"
+
+        .export         psg_init
+
+psg_init:
+        clx
+        stx     $0800           ; Select channel
```

Comment:
That line belongs inside the loop (otherwise, the loop just repeats on the first channel).